fypy/pricing/Lattice/BinomialLattice.py

```diff
@@ -0,0 +1,61 @@
+import math
+
+
+def binomial_lattice_black_scholes(S_0: float,
+                                   K: float,
+                                   r: float,
+                                   T: float,
+                                   sigma: float,
+                                   M: int,
+                                   call: int,
```

Comment:
Why is call an int? It is not intuitive what int(s) should represent calls, and what should represent (presumably) puts, and according to your code below, literally any number other than 1 means put, which is odd. 
Perhaps use a boolean is_call, which forced two states, or create an enum for call and put.

Reply:
Good catch @nrupprecht, and this is consistent with our other pricing functions that take an array of bools, e.g. StrikesPricer:  :param is_calls: np.array[bool], indicators of if strikes are calls (true) or puts (false) ... @abhinav1630, please update when you can